.github/actions/publish_to_npm/action.yml

```diff
@@ -5,16 +5,12 @@ inputs:
 runs:
   using: composite
   steps:
-  # TODO: Figure out a way to specify whether to publish to "latest" or "next" tag
```

Comment:
This is resolved. Now our script would correctly assign the npm tag `latest` or `next`: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/88b26c1fcc81de25acb884e0cf77bc8063fb94b5/scripts/publish-to-npm.js#L68